### ISSUE TITLE
Withdraw screen not showing correct number of pooled tokens

### ISSUE THREAD

**Issue by lauracw100** [scrtlabs/EthereumBridgeFrontend#143]
I provided liquidity for this sSCRT/SEFI pair. The total tokens is > 17 but the individual tokens are showing an amount of "0". And pool share, although it's a small percentage, shouldn't say "0%" should it? It's probably pretty low but it can get confusing for users. Maybe just change it to "< 1%" or something similar if no decimal value is desired.

![image](https://user-images.githubusercontent.com/436905/115613512-79710980-a2ba-11eb-8fa3-4bab41e582d6.png)

**Comment by alejandrordzl** [comment]
The UI team reviewed the function used to display the percentage, we discovered that 0% is used as a default value so we suspect the backend is either not returning a value or returning error. 

We are turning this over to Enigma team for troubleshooting.

**Comment by cankisagun** [comment]
cc @Cashmaney @assafmo

**Comment by assafmo** [comment]
Yes, it seems that the backend does not return this pool's values, do you see any errors in the console or network tabs? @lauraweindorf is this still an issue for you?

**Comment by lauracw100** [comment]
yeah, still seeing an issue after providing liquidity and then going to the Withdraw screen. The pooled SEFI and sSCRT plus my pool share are all showing 0.

![image](https://user-images.githubusercontent.com/436905/116285392-110fa580-a75c-11eb-988c-4d19414fdc9a.png)

**Comment by cankisagun** [comment]
@assafmo cc @Cashmaney it seems like this a backend issue that we should solve. We also have the same problem in issue #142 

We have the new UI launch scheduled for Friday at the moment. Is this something we can look at before?

**Comment by cankisagun** [comment]
@David-Sod pls see the same comment in #142 - these work fine in the current UI, you must have missed something
![image](https://user-images.githubusercontent.com/15679491/116314412-ef1a2100-a763-11eb-9b61-ffc799533186.png)
![Uploading image.png…]()

**Comment by alejandrordzl** [comment]
@lauraweindorf  @cankisagun .Yes, this was related to issue #142. This issue was solved with yesterday's commit, moving this to Done
